OSDOCS-2883: add balancing algorithm known issue (#38405)
openshift/openshift-docs: release_notes/ocp-4-8-release-notes.adoc

* Following an {product-title} single node reboot, all pods are restarted which causes significant load and longer than normal pod creation times. This happens because the Container Network Interface (CNI) is not able to process the `pod add` events quickly enough. The following error message is displayed: `timed out waiting for OVS port binding`. The {product-title} single node instance eventually recovers, though more slowly than expected. This known issue applies to the {product-title} version 4.8.15 and later. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986216[*BZ#1986216*])

> * Prior to {product-title} 4.8 the default balancing algorithm was `leastconn`. The default was changed to `random` in {product-title} 4.8.0 for non-passthrough routes. Switching to `random` increases memory consumption for every HAProxy process, and the cumulative memory usage is significant if you have several websocket connections. To mitigate this significant memory consumption, the default balancing algorithm was reverted to `leastconn` in {product-title} 4.8. Once there is a solution that does not incur significant memory requirements the default will be changed to `random` in a future {product-title} release.

Reviewer: ```suggestion
* Prior to {product-title} 4.8 the default load balancing algorithm was `leastconn`. The default was changed to `random` in {product-title} 4.8.0 for non-passthrough routes. Switching to `random` is incompatible with environments that need to use long-running websocket connections because it significantly increases memory consumption in those environments.  To mitigate this significant memory consumption, the default load balancing algorithm was reverted to `leastconn` in {product-title} 4.8. Once there is a solution that does not incur significant memory usage, the default will be changed to `random` in a future {product-title} release.
```

Author: Updated, thank you!

+
You can check the default setting by entering the following command:
+
[source,terminal]
----
$ oc get deployment -n openshift-ingress router-default -o yaml | grep -A 2 ROUTER_LOAD_BALANCE_ALGORITHM
        - name: ROUTER_LOAD_BALANCE_ALGORITHM
          value: leastconn
----
+

> The `random` option is still available, but routes that would benefit from this algorithmic choice must explicitly request that by setting the following annotation on a per-route basis by entering the following command:

Reviewer 2: ```suggestion
The `random` option is still available. However routes that would benefit from this algorithmic choice must explicitly set that option in an annotation on a per-route basis by entering the following command:
```

Reviewer 2: Maybe this instead to avoid two "by"s in one sentence?
